Source/ThirdParty/cronosplay/lib/Win64/.gitattributes

```diff
@@ -0,0 +1,4 @@
+play_cpp_sdk.dll filter=lfs diff=lfs merge=lfs -text
+cxxbridge1.lib filter=lfs diff=lfs merge=lfs -text
+play_cpp_sdk.dll.lib filter=lfs diff=lfs merge=lfs -text
+play_cpp_sdk.lib filter=lfs diff=lfs merge=lfs -text
```

Comment:
if this are put directly in the repo, there should be a CI check to verify their checksum matches the release

Reply:
i'll check to ensure checksum

Reply:
Including the libraries makes cloning slow and repo big, and difficult to verify them valid or not (if someone injects malicious codes in it). Seems no big benefits... Please reconsider that.

Reply:
Besides their maybe many in the future, android, linux, ios, etc. It is difficult to include them all.

Reply:
ok,removed binaries
we can release as current paradigm.